demo/api/src/db/migrations/Migration20240516085053.ts

```diff
@@ -0,0 +1,9 @@
+import { Migration } from '@mikro-orm/migrations';
+
+export class Migration20240516085053 extends Migration {
+
+  async up(): Promise<void> {
+    this.addSql('alter table "Manufacturer" add column "name" varchar(255) not null;');
```

Comment:
This migration will fail for existing rows. Please add a `truncate "Manufacturer"` to delete all rows before adding the new column.